html-forms/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <title>
+    HTML Forms
+  </title>
+  <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+  <h1>
+    Coding Form!
+  </h1>
+  <form action="http://www.example.com/review.php" method="get">
+    <label for="e-mail">E-Mail</label>
+    <div>
+      <input type="text" id="e-mail" name="e-mail" class="e-mail">
+    </div>
+    <label for="code-question">Why is coding cool?</label>
+    <div>
+      <select name="code-question" id="code-question">
+        <option value="interesting">Because it's interesting</option>
+        <option value="fun">Because it's fun</option>
+        <option value="powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <p>
+      <label for="favorite-language">Favorite Language &lpar;choose one&rpar;</label>
+    </p>
+    <input type="radio" id="html" name="favorite-language" value="html" checked="checked"> HTML
```

Comment:
you need a label for each of these with match for and id attributes, or you can wrap the inputs with the label